README.md

```diff
@@ -8,6 +8,17 @@ This extension is configured to talk to a language server over WebSockets. If yo
 private Sourcegraph instance, you should run your own language server. The server is available as a
 Docker image `sourcegraph/lang-typescript` from Docker Hub.
 
+### 🚨 Secure deployment 🚨
```

Comment:
I wouldn’t use these emojis - the purpose of this section I assume is to give the user confidence their deployment is secure, but this creates more of an alarming feeling / panic. Maybe use 🔐 instead.

Reply:
:+1: Switched to 🔐